### PR TITLE
Accessibility update

### PR DESCRIPTION
**Pull request by KaskMIL**
# In this new Pull Request I update my portfolio with Accessibility changes on the Index and CSS file
- **Page Title**
No issues

- **Image text alternatives**
Edit AngelList and Medium icons, and images on About-section for a better alternative text.

- **Text headings**
Change heading in Intro-section.

- **Colour Contrast**
No issues found

- **Resize**
No issues found

- **Interaction**
Added aria-labels to form inputs

- **Moving Content**
No issues found

- **Multimedia**
No issues found

- **The basic structure of the page**
No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -225,13 +225,13 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
...
+                        <input aria-label="user name" type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
                     </li>
                     <li>
-                        <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                        <input aria-label="email" type="email" id="mail" name="user_email" placeholder="Email address" required>
                     </li>
                     <li>
-                        <textarea name="user_msg" id="msg" maxlength="500" placeholder="Enter text here"
+                        <textarea aria-label="message" name="user_msg" id="message" maxlength="500" placeholder="Enter text here"
                             required></textarea>
```
[the remark this answers is not in the document]
Comment (zieeco) [reply index.html:235]: - [ ] please, kindly fix this color contrast issue.
- Use this [chrome extetnsion](https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf?hl=en) to analyze and fix all your color contrast issue.
- your see the project button 
- Your get in touch button
- Get my resume button
- The background color of each project card
- The CSS class `main .work-section .card ul li`
- The icons next to `Languages`, `Frameworks` and `Skills`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team


Great work on improving your portfolio accessibility 👏 
You've done well implementing them, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (KaskMIL) [reply index.html:235]: Hi @zieeco,
Thanks for your review!
About the labels, is it not enough with the aria-label attribute?
I read [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) that I could replace it, but maybe I'm wrong.
Will wait for your reply.
Thanks again!
cheers

[the remark this answers is not in the document]
Comment (zieeco) [reply index.html:235]: just use the label
 instead

**Review by keroloussamy — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**